SAMPLES.md

```diff
@@ -0,0 +1,104 @@
+# Backchannel integration
+
+There are three ways you can receive commands from Direct Line and handled by your page.
+
+## Integrate with JavaScript
+
+```html
+<!DOCTYPE html>
+<html>
+  <body>
+    <div id="webchat"></div>
+    <script src="//cdn.botframework.com/.../botchat.js"></script>
+    <script>
+      window.WebChat.renderWebChat({
+        directLine: window.WebChat.createDirectLine({
+          token: '...'
```

Comment:
token or secret? If the value is token they need to get the token also somewhere.

Reply:
This is as-is on v3 today. We need to revisit them with service team later.